vanguard.py

```diff
@@ -0,0 +1,49 @@
+import sys
+from pymarc import Record, Field, MARCReader, MARCWriter
+
+#SILSLA-13
+
+def modify_5xx(record):
+	for old_field in range(500, 600):
+		if old_field != 590:
+			for fld in record.get_fields(old_field):
+				for sfld in fld.get_subfields('5'):
```

Comment:
The $5 is not supposed to be repeated
https://www.loc.gov/marc/bibliographic/bd700.html (and others)
so you can simplify this by just testing the $5 directly like `if fld['5'].startswith('CLU')`

That would simplify most of these methods.